docs/gulp.md

```diff
@@ -77,4 +78,9 @@ The `fix` command runs the syntax fixers provided by the code linters used for
 validating SCSS and JavaScript (ES6). The command will try its _best_ to fix
 coding standard violations based on specific rulesets for each language.
 
+## Optimize command
+The `optimize` command will run through the theme images directory
+`web/themes/custom/**/img/*` and reduce file size for all `.gif` `.jpeg` `.png`
```

Comment:
Comma's between each file extension

Reply:
something to note this folder could be `img` or `images` depending on the project.